wave_matching/tests/ndt_tests.cpp

```diff
@@ -53,7 +53,7 @@ TEST_F(NDTTest, fullResNullMatch) {
     match_success = matcher->match();
     double diff = (matcher->getResult().matrix() - perturb.matrix()).norm();
     EXPECT_TRUE(match_success);
-    EXPECT_LT(diff, 0.1);
+    EXPECT_LT(diff, 0.12);
```

Comment:
For these comparisons it's as if the operator is inserted between the arguments. 
EXPECT_LT( val1, val2 ) checks if val1 < val2. ([gtest](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md))